src/swap/useSwapQuote.ts

```diff
@@ -23,19 +23,32 @@ const DECREASED_SWAP_AMOUNT_GAS_FEE_MULTIPLIER = 1.2
 
 export const NO_QUOTE_ERROR_MESSAGE = 'No quote available'
 
-export interface QuoteResult {
+interface BaseQuoteResult {
+  swapType: 'same-chain' | 'cross-chain'
   toTokenId: string
   fromTokenId: string
   swapAmount: BigNumber
   price: string
-  appFeePercentageIncludedInPrice: string | undefined
   provider: string
   estimatedPriceImpact: string | null
-  allowanceTarget: string
   preparedTransactions: PreparedTransactionsResult
   receivedAt: number
+  allowanceTarget: string
+  appFeePercentageIncludedInPrice: string | undefined
+  sellAmount: string
 }
 
+interface SameChainQuoteResult extends BaseQuoteResult {
+  swapType: 'same-chain'
+}
+
+interface CrossChainQuoteResult extends BaseQuoteResult {
+  swapType: 'cross-chain'
+  maxCrossChainFee: string
```

Comment:
there are two more [cross-chain properties](https://github.com/valora-inc/valora-rest-api/blob/c58f6257fb63ce53365cf2240de43a96226d295b/src/swaps/providers/squid.ts#L212-L214) returned from the API. perhaps we can list them here already. wdyt?
* `estimatedDuration` (in seconds)
* `estimatedCrossChainFee` (slightly reduced max fee to not scare the user)